nixos/modules/services/web-apps/whitebophir.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.whitebophir;
+in {
+  options = {
+    services.whitebophir = {
+      enable = mkOption {
```

Comment:
`mkEnableOption` might be nice here.

Reply:
Thanks for the pointer! Fixed in next push.